source/features/follow-file-renames.tsx

```diff
@@ -5,45 +5,48 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import * as api from '../github-helpers/api';
 import parseRoute from '../github-helpers/parse-route';
+import {getRepoURL} from '../github-helpers';
 
 interface File {
 	previous_filename: string;
 	filename: string;
 	status: string;
 }
 
-async function findRename(
-	user: string,
-	repo: string,
-	lastCommitOnPage: string
-): Promise<File[]> {
+async function findRename(lastCommitOnPage: string): Promise<File[]> {
 	// API v4 doesn't support it: https://github.community/t5/GitHub-API-Development-and/What-is-the-corresponding-object-in-GraphQL-API-v4-for-patch/m-p/14502?collapse_discussion=true&filter=location&location=board:api&q=files%20changed%20commit&search_type=thread
-	const {files} = await api.v3(`repos/${user}/${repo}/commits/${lastCommitOnPage}`);
+	const {files} = await api.v3(`repos/${getRepoURL()}/commits/${lastCommitOnPage}`);
 	return files;
 }
 
 function init(): false | void {
 	const disabledPagination = select.all('.paginate-container [disabled], .paginate-container .disabled');
+	const parts = parseRoute(location.pathname);
```

Comment:
Let's use consistent naming everywhere parseRoute is used
```suggestion
	const pathnameParts = parseRoute(location.pathname);
```

Reply:
> Everywhere

Reply:
>That's right! I'd name the variables just `path` or now instead of `pathnameParts`
>
>So you can use them as `path.branch`, `path.user`, etc
>
>_Originally posted by @fregante in https://github.com/sindresorhus/refined-github/pull/3109#issuecomment-631038563_